docs/Performance.md

```diff
@@ -0,0 +1,242 @@
+# Protobuf Perforamcne
+This benchmark result is tested on workstation with processor of Intel® Xeon® Processor E5-2630 and 32GB RAM
+
+## Parsing performance
+<table>
+<tbody><tr>
+<th rowspan="2"> </th>
+<th colspan="6" rowspan="1">cpp</th>
+<th colspan="6" rowspan="1">java</th>
+<th colspan="6" rowspan="1">python</th>
+</tr>
+<tr>
+<th colspan="2">new</th>
+<th colspan="2">newarena</th>
+<th colspan="2">reuse</th>
+<th colspan="2">deserializeFromByteArray</th>
+<th colspan="2">deserializeFromByteString</th>
+<th colspan="2">deserializeFromMemoryStream</th>
+<th colspan="2">cpp-generated-code</th>
+<th colspan="2">cpp-reflection</th>
+<th colspan="2">pure-python</th>
+</tr>
+<tr>
+<td>google_message1_proto2</td>
+<td>589ns</td><td>368.717MB/s</td>
```

Comment:
Remove the "xxx ns" column. That number isn't useful.

Reply:
Done